next/MultipleContents/README.md

```diff
@@ -0,0 +1,79 @@
+# Multiple Contents
+
+This sample demonstrates usage of the [`3DTILES_multiple_contents`](https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_multiple_contents) extension for storing multiple contents objects in a single tile.
+
+![MultipleContents](screenshot/MultipleContents.gif)
+
+The root tile of the tileset contains a model, namely a simple plane with sine waves, in two different representations:
+
+- A low-resolution representation as a textured triangle mesh glTF asset
+- A high-resolution point-grid representation with vertex colors, as another glTF asset
+
+The following sandcastle code renders this example tileset when it is hosted on a local server, and allows toggling the visibility of both representations:
+
+```
+// Enable experimental features for 3DTILES_multiple_contents support
+Cesium.ExperimentalFeatures.enableModelExperimental = true;
+
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+// Flags that are connected to the Sandcastle toggle buttons,
+// to show and hide the triangle- and point based content
+var showTriangles = true;
+var showPoints = true;
+
+// The two different contents from the root tile. These are
+// read in the 'tileLoad' event listener, and stored here
+// for access from the Sandcastle UI
+var trianglesContent;
+var pointsContent;
+
+// Load the tileset from a local server
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/tileset.json'
+}));
```

Comment:
Simplified the tileset setup code a bit and pointed out some CesiumJS idioms:

- moved the model matrix construction to the tileset creation time
- Used `Cartesian3.fromDegrees()` which is simpler than `Cartographic.toCartesian(Cartographic.fromDegrees())`
- moved the model a little bit into the air, it was clipping through the terrain by a little bit.
- `viewer.zoomTo()` can be called as soon as the tileset is constructed as I mentioned in another comment.

```suggestion
// Position the tileset on the globe.
var position = Cesium.Cartesian3.fromDegrees(-75.1596759, 39.9509025, 0.25);
var modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(position);

// Load the tileset from a local server
var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
    url : 'http://localhost:8003/tileset.json',
    modelMatrix : modelMatrix
}));
viewer.zoomTo(tileset);
```